inference-engine/src/transformations/src/transformations/common_optimizations/eliminate_unsqueeze_gather.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/eliminate_unsqueeze_gather.hpp"
+
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::EliminateUnsqueezeGather, "EliminateUnsqueezeGather", 0);
+
+ngraph::pass::EliminateUnsqueezeGather::EliminateUnsqueezeGather() {
+    const auto unsqueezeAxis = ngraph::pattern::any_input();
```

Comment:
Please add `MATCHER_SCOPE` marco. It is needed for conditional compilation feature.
Examples you can find in the PR https://github.com/openvinotoolkit/openvino/pull/3795

Reply:
Thanks! Done.